### PR TITLE
Add FeedbackFruits platform description and extensibility

### PR DESCRIPTION
**Pull request by steffansluis**
This PR add a description of the current FeedbackFruits platform, the technologies it builds on and the limitations this imposes on any solution to the problem.

### CODE REVIEW

**Comment by steffansluis** [comment]
@bartist @WouterKvG Please review!

File: feedbackfruits/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Integration with FeedbackFruits
+
+To integrate with the existing FeedbackFruits platform, some requirements have to be met by any solution to the problem. This section lists these requirements, and offers some suggestions for meeting them.
+
+## Current platform structure
+
+FeedbackFruits is an online education platform, built on modern web technologies such as Ruby on Rails [^rubyonrails], Ember.js [^emberjs], MongoDB [^mongodb] and Node.js [^nodejs]. The platform is structured with micro-services in the form of different servers that each serve a single front-end, and are also capable of talking to each other.
```
Comment (WouterKvG) [inline feedbackfruits/README.md]: `that each serve a single front-end` _each_ should be removed I think.

**Comment by WouterKvG** [comment]
Maybe it is in a good idea to add a graph to the current platform structure to illustrate how the major components are wired perhaps with how the extensions can be fit in.

File: feedbackfruits/README.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+The platform is heavily structured around activities, such as videos, documents and assignments. These activities have an API and can be extended to incorporate custom functionality. A Ruby on Rails back-end aggregates the different engines that offer types of activities and serves it as one API.
+
+The activities get requested in an Ember.js front-end and displayed in a material design [^materialdesign] user interface. Each type of activity can optionally be rendered using custom logic for that type, contained within an Ember addon [^emberaddon].
+
+## Extensibility
+
+At the moment of writing this report, there are two ways to integrate with the FeedbackFruits platform in a modular way. The first and simpler way is to add a new type of activity. This can be done by creating a Rails engine [^railsengine] and implementing the activity according to the existing API. This will allow the engine to run within the FeedbackFruits platform. The activity can then be displayed in the frontend with an Ember addon for the new activity type.
```
Comment (WouterKvG) [inline feedbackfruits/README.md]: `two ways to integrate ... in a modular way` is a bit weird to read.

**Comment by steffansluis** [comment]
@WouterKvG I agree about the graph of the current platform, but I think we should merge this anyway and add it later.

**Pull request merged**